cpp/daal/include/data_management/data/csr_numeric_table.h

```diff
@@ -94,6 +94,9 @@ class DAAL_EXPORT CSRBlockDescriptor
         return _rows_ptr.get();
     }
 
+    /**
+     * \DAAL_DEPRECATED
+     */
     inline daal::services::internal::Buffer<DataType> getBlockValuesBuffer() const
```

Comment:
Just a question, should we add DAAL_DEPRECATED  before some of the methods? Or what plan we have to remove/modify deprecated functions?